translations/strings.yaml

```diff
@@ -82,3 +98,172 @@ splittunnel:
+    privacyNoticeLinkText: Mozilla VPN Privacy Notice
+    primaryButtonText: Submit
+    secondaryActionText: Cancel
+    #screen MacOS / 31
+    dropdownListItem: Payment and billing
+    dropdownListItem: Account issues
+    dropdownListItem: Technical issues
+    dropdownListItem: Request features
+    dropdownListItem: Other
+    #screen MacOS 32
+    responseHeader: Thank you!
+    responseBody: Our support team will contact you at _email_ once we've been able to look at your issue.
+    responseButton: Done
+    #error state screen MacOS 39
+    errorHeader: Error submitting your support request
+    errorText: An unexpected error has occurred, please try again.
+    errorButton: Try Again
+
+#v2.5 What's New? Release notes
+whatsNewReleaseNotes:
+  #macOS tour modal alert on system update
+  #screen macOS / 92
+  modalHeader: What's new in VPN?
+  modalBodyText: We've added a ton of cool features in this latest release. Take the tour to get a deeper dive into what we've added for you!
+  modalButtonText: Take the tour
+  #screen MacOS / 89
+  modalHeader: Multi-hop VPN
+  modalBodyText: Multi-hop VPN will route your traffic through a second server for added protection. You can find this feature on the "Select location" screen.
+  modalButtonText: Next
+  #screen MacOS / 90
+  modalHeader: In-app support form
+  modalBodyText: The in-app support form will allow you to contact support from within the VPN app. You can find it in the "Get help" section.
+  modalButtonText: Next
+  #screen MacOS / 91
+  modalHeader: Custom DNS
+  modalBodyText: Custom DNS servers allow for faster speed using local networks, features like ad-blocking and anti-tracking. You can find this feature in the "Network settings" section.
+  modalButtonText: Done
+#macOS tour within settings
+  #screen macOS / 76
+  navLabel: What's new
+  #screen MacOS / 87
+  pageHeader: What's new
+  subHeader: Take the tour
+  listItemHeader: Multi-hop VPN
+  listItemBody: Multi-hop VPN uses multiple VPN servers instead of one for extra security and privacy.
+  listItemHeader: In-app support form
+  listItemBody: The in-app support form will streamline the process of getting your issues resolved.
+  listItemHeader: Custom DNS
+  listItemBody: Custom DNS settings allow for faster connection using local servers.
+  #system notifications for all platforms
+  notifHeader: What's new in VPN
+  notifBody: Check out all the new features available in the latest version of Mozilla VPN!
+  notifCTA: Take the tour
+
+#In-app purchase strings for variable pricing
+inAppPurchaseVariable:
+  #screen VPP_Android_00
+  getHelpLink: Get help
+  pageHeader: Mozilla VPN
+  featureListItem: No logging of your network activity
+  featureListItem: No one can track you
+  featureListItem: 280+ servers in 30+ countries
+  featureListItem: Connect up to 5 devices
+  annualPlanCardHeader: $4.99/month
```

Comment:
This should not be hardcoded with US values, same for all the next strings.

@bakulf 
What's the expectation for this type of strings on your side? I don't think content has visibility on how the code should look or behave.

Reply:
All of these strings should be removed because they are already implemented for iOS.
See: vpn.subscription.price

Reply:
Remove from line 225 to 248.